packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -189,7 +192,20 @@ export class DebugConfigurationManager {
         }
     }
     async addConfiguration(): Promise<void> {
-        const { model } = this;
+        let model: DebugConfigurationModel | undefined;
+        if (this.models.size > 1) {
+            model = await this.quickPick.show(Array.from(this.models.entries()).map(entry => {
+                const [rootFolder, modelValue] = entry;
```

Comment:
It would be nice if the list of roots would be ordered like what the user sees in the explorer: I guess if we iterate over the root folders array and 'get' from the map that would be the case. Iteration order over a map is not defined.

Reply:
Thank you for the feedback !
With my most recent update, I am using `workspaceService.roots` to ensure the order of the workspace folders.